sqlserver.go

```diff
@@ -125,3 +125,11 @@ func (h *SQLServer) disableReferentialIntegrity(db *sql.DB, loadFn loadFunction)
 
 	return tx.Commit()
 }
+
+// Splitter is a BatchSplitter interface implementation. We need it for
+// SQL Server because commands like a `CREATE SCHEMA...` and a `CREATE TABLE...`
+// could not be executed in the same batch.
+// See https://docs.microsoft.com/en-us/previous-versions/sql/sql-server-2008-r2/ms175502(v=sql.105)#rules-for-using-batches
```

Comment:
Ouch!

I think I should move the schema files to either a function or an YAML file (so arrays are allowed to run multiple statements) to prevent logic like this

But for this PR this is good enough as is, we can do that later